docs/home/whats-new-2-15.md

```diff
@@ -1,94 +1,103 @@
+* Added the ability to configure strict GPU compute time slicing. This gives workloads the exact GPU compute portion based on the requested GPU fraction (GPU Memory Fraction). This creates complete transparency and predictability of the amount of resources (Compute, Memory, etc.) a workload will get from a GPU. For more information, see [GPU Time Slicing](##gpu-time-slicing).
 
-<!-- RUN-10619 Email mechanism -->
+<!-- RUN-10862/RUN-10863 Department as a workspace phase 2 - scope in credentials -->
+* Improved *Credentials* creation. Now, a Run:ai scope can be added to credentials. For more information, see [Credentials](../admin/admin-ui-setup/credentials-setup.md).
 
-<!-- RUN-10622 Policy blocks workloads that attempt to store data on the node-->
+<!-- RUN-10271/RUN-10321 Mark environment for workload type-->
+* Added support for workload types when creating a new or editing and existing environment. Select from `single-node` or `multi-node (distributed)` workloads. The environment is available only on feature forms which are relevant to the workload type selected.
 
-<!-- RUN-10639/RUN-11389 Researcher Service Refactoring - Step 1 \(UI only\)-->
+<!-- RUN-10639/RUN-11389 - Researcher Service Refactoring -->
+* Improved configuration for authentication in the cluster. Configuring the Kubernetes cluster API server for authentication is no longer needed. This applies to environments where end users are using the UI only and not the CLI.
 
-<!-- RUN-10802 Quota management dashboard post-release -->
+<!-- RUN-12505/RUN-12506 - Support Kubeflow notebooks for scheduling/orchestration -->
+* Improved support for KubeFlow Notebooks. Now Run:ai supports scheduling of Kubeflow notebook CRDs with fractional GPUs. Kubeflow notebooks are identified automatically and use a special icon in the *Jobs* UI.
 
-<!-- RUN-10862/RUN-10863 Department as a workspace asset creation scope - phase 2-->
+<!-- RUN-10251/RUN-10252 - Block over-subscription of quota by Projects/Departments- -->
+* Improved quota settings by providing the ability to block over-subscription of quota by *Projects* or *Departments*. For more information, see [Over quota blocking](../Researcher/scheduling/the-runai-scheduler.md#limit-quota-over-or-under-subscription).
 
-<!-- RUN-11186/RUN-11241 Submitting an MPI/PT/TF/XGBoost distributed training from UI - 1st form page -->
+<!-- RUN-10404/RUN-11747 Submit distributed training
+* RUN-11194/RUN-11239 All changes done in the UI for distributed training are hidden behind feature flag 
+* RUN-11186/RUN-11241 Submitting an MPI/PT/TF/XGBoost distributed training from UI - 1st form page 
+* RUN-11206/RUN-11242 Submitting distributed training from UI - 2nd form page
+* RUN-11219/RUN-11602 Submitting an MPI distributed training from UI - 3rd form page
+* RUN-11231/RUN-11240 Environment for distributed training
+-->
+* Added support for distributed training in the following:
```

Comment:
Yes it was added in these forms. But important to give context before. Added support to run distributed workloads via Run:ai workspaces & training by adding support...to the following forms / assets /